docs/intro.rst

```diff
@@ -60,3 +60,4 @@ dependencies using conda then install ``odc-stac`` with pip. Sample
      - pip =20
      - pip:
        - git+https://github.com/opendatacube/odc-stac/
+       - pystac-client
```

Comment:
as before, `pystac-client` should be installed from conda not pip

Reply:
Done